db/migrate/20141223142759_fill_discovery_attribute_sets_for_existing_hosts.rb

```diff
@@ -1,6 +1,20 @@
+class FakeDiscoveredHost < ActiveRecord::Base
+  self.table_name = 'hosts'
+end
+
 class FillDiscoveryAttributeSetsForExistingHosts < ActiveRecord::Migration
   def up
-    Host::Discovered.all.each { |host| host.populate_fields_from_facts}
+    FakeDiscoveredHost.where(:type => "Host::Discovered").all.each do |host|
+      begin
+        say "Populating attribute set for discovered host #{host.name}"
+        host.discovery_attribute_set = DiscoveryAttributeSet.where(:host_id => host.id).first_or_create
```

Comment:
These 2 rows are identical to what we have in `populate_fields_from_facts`, maybe there should be a method on the host for populating the attribute set in case we ever change it.